readme.md

```diff
@@ -299,6 +299,7 @@
 - [Web Performance Budget](https://github.com/pajaydev/awesome-performance-budget#readme) - Techniques to ensure certain performance metrics for a website.
 - [Web Animation](https://github.com/sergey-pimenov/awesome-web-animation#readme) - Animations in the browser with JavaScript, CSS, SVG, etc.
 - [Yew](https://github.com/jetli/awesome-yew#readme) - Rust framework inspired by Elm and React for creating multi-threaded frontend web apps with WebAssembly.
+- [StimulusJS](https://github.com/skatkov/awesome-stimulusjs#readme) - A modest JavaScript framework for the HTML you already have.
```

Comment:
The description should succinctly describe what it actually is, not be a marketing tagline.